puppet/modules/jenkins_node/manifests/packaging/debian.pp

```diff
@@ -37,7 +37,7 @@
       release    => 'bookworm',
       apturl     => $debian_mirror,
       aptcontent => "deb ${debian_mirror} bookworm main non-free contrib\ndeb-src ${debian_mirror} bullseye main non-free contrib\n",
-      nodesource => false;
+      nodesource => true;
```

Comment:
technically, it's `true` by default, so we could even drop that line, on the other hand… better explicit?

Reply:
I'd rather be explicit here